cmd/cluster_kubeconfig.go

```diff
@@ -1,18 +1,25 @@
 package cmd
 
 import (
-	"bufio"
 	"errors"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"log"
 	"os"
-	"os/user"
+	"path"
 	"strings"
+	"time"
 
 	"github.com/spf13/cobra"
 	"github.com/xetys/hetzner-kube/pkg/clustermanager"
 	"github.com/xetys/hetzner-kube/pkg/hetzner"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	defaultContext = "kubernetes-admin@kubernetes"
```

Comment:
I'm not sure on why we are defining this `defaultContext`, maybe context should be "cluster specific", like `kubernetes-admin@{CLUSTER_NAME}`?

Reply:
this is not what we going to set, this is what we looking to rename. regardless cluster, this context set by default during kubebootstrap, so all our configs with this context. maybe i should change name to `default_pattern` then.